app/config/mandatee-table-config.js

```diff
@@ -1670,20 +1633,12 @@ export const mandateeTableConfigRMW = (meeting) => {
             }
             ?bestuursorgaan besluit:bestuurt <${bestuurseenheid.uri}>.
 
-            {
-              SELECT ?fractie (COUNT(?_persoon) AS ?fractie_grootte)
-              WHERE {
-                ?_mandataris a mandaat:Mandataris.
-                ?_mandataris org:hasMembership/org:organisation ?fractie.
-                ?_mandataris org:holds ?_mandaat.
-                ?_mandaat org:role <${BESTUURSFUNCTIE_CODES.GEMEENTERAADSLID}>.
```

Comment:
NOTE! This was using `BESTUURSFUNCTIE_CODES.GEMEENTERAADSLID` for an RMW query. I corrected this to `LID_BCSD`. Please verify if this is correct.